### ISSUE TITLE
In the link displayed link-status-redux, the letter "g" is trimmed from below

### ISSUE THREAD

**Issue by PhoenixIndigo** [jaatroko/link-status-redux#44]
In the link displayed link-status-redux, the letter "g" is trimmed from below. The status panel hides the extension of the URL Tooltip We.
![link-status-redux](https://github.com/jaatroko/link-status-redux/assets/107413021/1272f701-076d-454d-8a84-4651fbf8a996)
![link-status-panel](https://github.com/jaatroko/link-status-redux/assets/107413021/917fe421-2504-4655-9771-7e3adf42d6e8)

**Comment by jaatroko** [comment]
This affects only the URL part, not the prefix (indicator) or postfix (visit time). This is due to CSS overflow set to hidden for the URL part: the space is not quite big enough to contain the full height of the text, resulting in clipped descenders. Prefix and postfix are fine because they are allowed to overflow on top of the panel padding.

The fix is non-trivial, I will have to investigate and think about how to best fix this.

**Comment by jaatroko** [comment]
Fixed in version 3.19 available from AMO. The fix is a bit brute-force as it just increases the div and line height a bit, so it is not *guaranteed* to work with every font and font size, but it did work for my Windows and Linux tets cases.

**Comment by PhoenixIndigo** [comment]
It turned out that with a zoom from 133% to 450% the problem is still observed:
![](https://github.com/jaatroko/link-status-redux/assets/107413021/03f4954b-8d7c-446f-b40a-d17420167685)

**Comment by jaatroko** [comment]
Just adding a note here to say that this issue has not been *completely* forgotten.

The idea I had was to load the overlay with a dummy placeholder text which gets (briefly) rendered, and get the font height from that. Upon further thought, the problem is that you cannot use a placeholder text due to font substitution: the actual URL text can have characters that the placeholder does not (e.g. CJK characters), which may get rendered with a different font (because the "normal" font does not have those glyphs) which can have a different height. This effectively means that you cannot know the dimensions of a rendered text string until *that exact string* has actually been rendered. And here we would use the height information to affect *how* the text is rendered, so it is kind of a chicken-or-egg problem, and if there is a solution, it is not simple or easy.

In short: there (probably) will not be any further fixes for this issue.
